test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
```

Comment:
While it is nice to have the test code near what it tests, that differs pretty significantly from our usual style.  Maybe consider moving this and the .good file under a new folder test/modules/standard/jama?  That will allow us to run it nightly